go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/gadelkareem/que
```

Comment:
This prefix prob needs to be changed. Curious, why the name change from `que-go` to `que`? Guessing this won't impact existing code still pointing to `github.com/bgentry/que-go`?